src/components/outline/DraggableTree.tsx

```diff
@@ -277,13 +277,15 @@ export const DraggableTree: React.FC<DraggableTreeProps> = ({
 
           <OutlineItemLink to={`#${node.attrs.id}`}>
             <OutlineItemIcon>{nodeTypeIcon(node.type)}</OutlineItemIcon>
-            <OutlineItemLinkText
-              className={`outline-text-${node.type.name} ${
-                isDeletedItem && 'deleted'
-              }`}
-            >
-              {itemText(node)}
-            </OutlineItemLinkText>
+            {node.type.name != 'manuscript' && (
```

Comment:
Similarly, you need to remove the entire outline item, including the arrow and the icon, not just the text.

Reply:
@IslamJMomani, done, sorry for the confusing.